src/core/Akka.Remote.Tests/RemotingTerminatorSpecs.cs

```diff
@@ -102,5 +101,40 @@ public void RemotingTerminator_should_shutdown_properly_with_remotely_deployed_a
                 Assert.True(terminationTask.Wait(RemainingOrDefault), "Expected to terminate within 10 seconds, but didn't.");
             });
         }
+
+        [Fact]
+        public void RemotingTerminator_should_shutdown_properly_without_exception_logging_while_graceful_shutdown()
```

Comment:
The body of this test is taken from the this same spec - RemotingTerminator_should_shutdown_properly_with_remotely_deployed_actor